experimenter/experimenter/targeting/constants.py

```diff
@@ -1764,6 +1764,17 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+IOS_IPHONE_USERS_ONLY = NimbusTargetingConfig(
+    name="iPhone users only",
+    slug="ios_iphone_users_only",
+    description="Targeting iPhone users",
+    targeting="is_phone",
+    desktop_telemetry="",
+    sticky_required=True,
```

Comment:
You shouldn't need sticky here since a profile on an iphone can't be migrated onto an ipad or other ios device, I don't think?